runner/llamarunner/runner.go

```diff
@@ -1,4 +1,4 @@
-package runner
+package llamarunner
```

Comment:
More of a nit comment, but why move this to `runner/llamarunner` vs keeping it in `llama/runner`?

Reply:
Since the two runners have parallel structure, share code and have a common entry point, it's more logical to me to group them together. The `llama` directory is mostly just C++ code and a Go wrapper at this point so it's not even all that related to the runner.

If I were to have a different directory structure, I think would actually move `llama` under `runner/llamarunner` rather than the other way around. However, I'm not sure if that's actually better than the current one.